src/frontend/next/src/components/Post/Post.tsx

```diff
@@ -0,0 +1,210 @@
+import React, { useRef, useState } from 'react';
+
+import useSWR from 'swr';
+import 'highlight.js/styles/github.css';
+import { makeStyles } from '@material-ui/core/styles';
+import { Box, Grid, Typography, ListSubheader, createStyles } from '@material-ui/core';
+
+// For style, need to add:
+// import '../styles/telescope-post-content.css';
```

Comment:
Typo: This statement needs to be added to `index.tsx` to work properly.

Reply:
cc @tonyvugithub, can you take over the global styling we need?  We should get an issue filed to move the telescope-post-content.css file into our global css for next.js.

Reply:
@humphd : I believe @PedroFonsecaDEV is currently working on the global.css file in [#1548](https://github.com/Seneca-CDOT/telescope/issues/1548)

Reply:
Also can we convert the css file of Post to JSS to keep it consistent with the rest of the components? And then we don't have to worry about moving the css to global.css

Reply:
I don't think we can, or we can't do it easily.

The way we load and display posts now is to send raw HTML and inject it as `innerHTML`.  This means we can't rely on module scoped CSS for a component, since it isn't really a component.

It might be possible down the road to use something like [Server Components](https://reactjs.org/blog/2020/12/21/data-fetching-with-react-server-components.html), but they are still very new.  https://addyosmani.com/blog/react-server-components/ has a good discussion.  This is a research problem we could tackle on the way to 2.0.

For now, we should probably port it as is.

Reply:
OK, left a comment in #1548.

Reply:
You can remove this comment now, @PedroFonsecaDEV is doing it in another PR in the global stylesheet.